common/common.scss

```diff
@@ -1,214 +1,182 @@
+@import "badges_page";
 $primary: #162d71;
 $text: #30323e;
 $shadow: 0 2px 6px rgba(0, 0, 0, 0.08);
-
 // Overwrite discourse styles
 input {
-  &:focus {
-    box-shadow: none !important;
-  }
+    &:focus {
+        box-shadow: none !important;
+    }
 }
 
 .d-header {
-  box-shadow: 0 0 5px lightgrey;
+    box-shadow: 0 0 5px lightgrey;
 }
 
 .drop-down-mode .d-header-icons .active .icon,
 .menu-panel,
 .user-menu .notifications .read {
-  background-color: #f4f4f4;
+    background-color: #f4f4f4;
 }
 
 .menu-panel {
-  box-shadow: $shadow;
+    box-shadow: $shadow;
 }
 
 .menu-panel li {
-  a {
-    color: $text;
-
-    &:hover {
-      background-color: $primary;
-      color: #fff;
-
-      svg,
-      span {
-        color: #fff;
-      }
+    a {
+        color: $text;
+        &:hover {
+            background-color: $primary;
+            color: #fff;
+            svg,
+            span {
+                color: #fff;
+            }
+        }
+    }
+    &.heading a:hover {
+        background-color: $primary;
+    }
+    &.category-link a:hover span {
+        color: inherit;
     }
-  }
-
-  &.heading a:hover {
-    background-color: $primary;
-  }
-
-  &.category-link a:hover span {
-    color: inherit;
-  }
 }
 
 .user-menu .notifications li {
-  &:hover {
-    background: $primary;
-
-    a {
-      color: #fff;
-
-      svg,
-      span {
-        color: #fff !important;
-      }
+    &:hover {
+        background: $primary;
+        a {
+            color: #fff;
+            svg,
+            span {
+                color: #fff !important;
+            }
+        }
     }
-  }
 }
 
 // button#create-topic {
 //   background: $primary;
 //   color: #fff;
-
 //   svg {
 //     color: #fff;
 //   }
 // }
-
 // .category-list {
 //   tbody .category {
 //     border-left: none;
 //   }
-
 //   .subcategories {
 //     display: flex;
 //     flex-direction: column;
 //     margin: 0.5rem 0;
-
 //     .subcategory {
 //       padding-top: 1rem;
 //     }
 //   }
 // }
```

Comment:
I know it's not introduced by your PR, but it would be helpful if we could remove unused stylings.